tests/features/language-negotiation-suffix.feature

```diff
@@ -53,3 +53,14 @@ Feature: Language negotiation by suffix.
     Then I should see the error message 'The suffix for Spanish, es, is not unique.'
     Then I should see the error message 'The suffix for French, es, is not unique.'
 
+  @front-page
+  Scenario: Language switcher should maintain front page alias in the links.
```

Comment:
I'm guessing this should go in a separate behat feature since it's a completely separate module?